addons/website_event_questions/views/event_question_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record model="ir.ui.view" id="event_question_view_form">

```

Comment:
Missing indentation + better to put id before model :)  (indentation valid for other view files also)